Source/DataSources/KmlDataSource.js

```diff
@@ -1688,11 +2213,14 @@ define([
      * @param {Object} [options] An object with the following properties:
      * @param {DefaultProxy} [options.proxy] A proxy to be used for loading external data.
      * @param {String} [options.sourceUri] Overrides the url to use for resolving relative links and other KML network features.
+     * @param {Camera} [options.camera] The camera that is used for viewRefreshModes and sending camera properties to network links.
+     * @param {Canvas} [options.canvas] The canvas that is used for sending viewer properties to network links.
```

Comment:
I'll add the deprecation messages. No we don't crash. I have test documents that work fine. We spit out warnings if you have an onStop refresh saying we can't refresh because there isn't a camera. All the URL replacing for camera position, etc  in the URL check for a camera if its needed to get the value.